brewchat/src/main/java/com/example/brewchat/ChatService.java

```diff
@@ -66,17 +66,25 @@ public ChatService(Context context, String appId, String authKey, String authSec
             chatService.addConnectionListener(this);
         }
 
+        QBAuth.createSession(new QBEntityCallbackImpl<QBSession>() {
+            @Override
+            public void onSuccess(QBSession session, Bundle params) {
+                Log.d(TAG, session + " " + params);
+            }
+
+            @Override
+            public void onError(List errors) {
+                Log.d(TAG, errors.toString());
+                new LoginActivity().error();
```

Comment:
Please also handle errors with events. We are using an event bust to decouple this service from other components.